tests/50federation/34room-backfill.pl

```diff
@@ -447,3 +447,113 @@
           });
       });
    };
+
+test "Outbound federation rejects backfill containing invalid JSON for events in room version 6",
+   requires => [ local_user_fixture(), $main::INBOUND_SERVER, federation_user_id_fixture() ],
+
+   do => sub {
+      my ( $user, $inbound_server, $creator_id ) = @_;
+
+      my $local_server_name = $inbound_server->server_name;
+      my $datastore         = $inbound_server->datastore;
+
+      my $room_alias = "#50fed-31backfill:$local_server_name";
```

Comment:
why 31?

(this alias needs to be unique, so you might want to consider how that is arranged)